app/store/stacking/stacking.reducer.ts

```diff
@@ -82,8 +82,14 @@ export const stackingSlice = createSlice({
       action: PayloadAction<CoreNodePoxResponse>
     ) => {
       state.initialRequestsComplete.poxInfo = true;
+      state.errors.poxInfo = false;
```

Comment:
A lot of state management! Looks okay to me, although not familiar with this code at all

Reply:
Yes this is a bit messy state mgmt to be honest, but it has at least been easy to add very explicit behiaviours